Library/Formula/grc.rb

```diff
@@ -52,4 +28,15 @@ def caveats; <<-EOS.undent
       source "`brew --prefix`/etc/grc.bashrc"
     EOS
   end
+
+  test do
+    require "open3"
+
+    # Test the 'ls' config
+    Open3.popen3("grcat", (share+"grc/conf.ls")) do |stdin, stdout, _|
```

Comment:
This can probably be simplified to use `pipe_output` instead. `grep` other formulae for usage.